app/cosmos/init.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs')
+const path = require('path')
+
+const initCosmos = async () => {
+  // cosmos emulator is very slow to start up, if getting eccon refused suggest increase time to wait for cosmos to start
+  try {
+    const databasesDir = path.join(__dirname, './databases')
+    const files = fs.readdirSync(databasesDir)
+
+    const createDatabases = files.map(async file => {
+      const databaseName = `${file.split('.')[0]}Database`
+      const db = require(`./databases/${file}`)
+      return db[databaseName]()
+    })
+
+    await Promise.all(createDatabases)
+  } catch (err) {
+    console.log(err)
```

Comment:
Error ignored, is this safe?

Reply:
Oversight, have updated to throw error if database cannot be created

Reply:
It still throws an error?